app/src/main/java/de/grobox/transportr/networks/PickTransportNetworkActivity.java

```diff
@@ -146,19 +193,53 @@ private void selectItem() {
 			}
 		}
 	}
+	
+	private HashMap<Region, List<Region>> getRegionsByRegion() {
```

Comment:
`getRegionsByRegion()` sounds a bit confusing. Maybe we can find a clearer name?

Reply:
could be `getSubRegionsByRegion`? But the functions wouldn't even be necessary when changing this whole iterations thing.

Reply:
Removed this function in the latest commit.